MoreSeamothUpgrades/MonoBehaviours/BetterDrillable.cs

```diff
@@ -402,7 +402,7 @@ public ItemsContainer GetStorageContainer(Vehicle veh, Pickupable pickupable)
             {
                 var seamoth = (SeaMoth)veh;
 
-                for(int i = 0; i < 4; i++)
+                for(int i = 0; i < 9; i++)
```

Comment:
Revert this: 
1. Storage slots are not allowed in the extra slots provided by the Slot Extender [link](https://github.com/Senna7608/SennaSubnauticaPlugins/blob/master/SlotExtender/Patchers/Equipment_Patcher.cs#L25)
2. Do not use a hard-coded number of maximum slots when trying to account for the Slot Extender as there is no guarantee that all players will have that mod installed

Reply:
Ah I see.